src/containers/Application/ApplicationPageWrapper.tsx

```diff
@@ -44,10 +57,42 @@ const ApplicationPageWrapper: React.FC = () => {
         push,
         replace,
       })
+
+      if (sectionCode && page)
+        setCurrentSection(templateSections.find(({ code }) => code === sectionCode))
+    }
+  }, [isApplicationLoaded, sectionCode, page])
+
+  useEffect(() => {
+    const progressStructure = buildProgressInApplication({
+      elementsState,
+      responses: responsesFullByCode,
+      templateSections,
+      isLinear: application?.isLinear as boolean,
+      currentSection: currentSection?.index as number,
+      currentPage: Number(page),
+    })
+    setProgressInApplication(progressStructure)
+    setProcessingValidation(false)
```

Comment:
Hmm I am not sure why `processingValidation` is required

Reply:
It was required to call the `ProgressBar` component when the structure have finish the creation. It doesn't seem to be a problem to remove it now, so I've removed [here](https://github.com/openmsupply/application-manager-web-app/pull/127/commits/2336794c474455557585527bfbfde6221859600e)

Reply:
thanks